### DISCUSSION TITLE
Text Alignmement does not work on version 11.0.0 Windows

### DISCUSSION THREAD

**Discussion by werlacher** [flameshot-org/flameshot#2443] [Q&A] [answered]
Good afternoon,
after install the version 11.0.0 in my windows system, the option to align the text does not apper in the lateral toolbar, 
what is missing in the configuration or it's just like that in Windows ?
thanks
Wilques
![image](https://user-images.githubusercontent.com/38436625/154686928-b0787800-f90f-4201-8995-f500b9153632.png)

**Comment by mmahmoudian** [comment]
@werlacher I don't exactly remember, but I think that was added after the v11.0.0 release. You can either wait for the next stable release or use our nightly builds (might be from time to time buggy but it updates every time we add something, so it has the latest and greatest stuff, but potentially bleeding edge):

https://flameshot.org/docs/installation/development-build/

**Comment by werlacher** [comment]
@mmahmoudian, good morning. the version you indicate, it's also without the possibility to change the text alignment.
I will wait the version 11.0.XX. probably it's not compiled yet.

**Reply by mmahmoudian** [reply]
I don't have Windows, but I tried the nightly builds to Linux (should follow the same exact procedure) and it has the alignments (image below). Would you please right-click on the tray icon, then click on About and copy the information there for me using the copy button in the About window)

![image](https://user-images.githubusercontent.com/390889/154933215-7b89c6e1-5183-445b-8593-e18a35700fa4.png)

**Reply by mmahmoudian** [reply]
I also realized that in the first screenshot you provided, you have not selected the Text tool. The sidebar will show the settings for the selected tool.

**Comment by werlacher** [comment, marked as answer]
@mmahmoudian, you are the MAN. 👍 
yes, it's there, based in the tool I select.
Thank you so much for the explanation.
